src/App.tsx

```diff
@@ -1,61 +1,132 @@
+import { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+function getUserByName(name: string): User | null {
+  const foundUser = usersFromServer.find(user => user.name === name);
+
+  return foundUser ?? null;
+}
```

Comment:
why do you need this method? is there not enough finding user by id? try to refactor your code